test/aqua/test_compute_min_eigenvalue.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test VQE """
+
+import unittest
+from test.aqua.aqua_test_case import QiskitAquaTestCase
+from qiskit import BasicAer
+
+from qiskit.aqua import QuantumInstance, aqua_globals
+from qiskit.aqua.operators import WeightedPauliOperator, MatrixOperator
+from qiskit.aqua.components.variational_forms import RY, RYRZ
+from qiskit.aqua.components.optimizers import L_BFGS_B, SPSA
+from qiskit.aqua.algorithms import VQE, NumPyMinimumEigensolver
+
+
+class TestComputeMinEigenvalue(QiskitAquaTestCase):
```

Comment:
We can add this but I think the VQE tests and NumpyMinEigsolver tests cover this an more already

Reply:
I agree, I think this should be deleted